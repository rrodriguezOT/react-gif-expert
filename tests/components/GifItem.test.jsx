import { render } from '@testing-library/react';
import { GifItem } from '../../src/components/GifItem';

describe('Prueba de <GifItem />', () => {
    const url = 'https://media.giphy.com/media/d2PqO9s7M9CNO/giphy.gif';
    const title = 'S01e01 GIF';
    test('ValidaciÃ³n del snapshot', () => {
        const { container } = render(<GifItem title={title} url={url} />);
        expect(container).toMatchSnapshot();
    });
});