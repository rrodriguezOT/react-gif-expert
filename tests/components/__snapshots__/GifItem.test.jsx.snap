// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de <GifItem /> Validaci√≥n del snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="S01e01 GIF"
      src="https://media.giphy.com/media/d2PqO9s7M9CNO/giphy.gif"
    />
    <p>
      S01e01 GIF
    </p>
  </div>
</div>
`;
